chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -27,9 +27,9 @@
 
 tftproot = node[:provisioner][:root]
 
-pxecfg_dir="#{tftproot}/discovery/pxelinux.cfg"
-pxecfg_default="#{tftproot}/discovery/pxelinux.cfg/default"
-uefi_dir="#{tftproot}/discovery"
+pxecfg_dir="#{tftproot}/discovery/bios/pxelinux.cfg"
+pxecfg_default="#{tftproot}/discovery/bios/pxelinux.cfg/default"
```

Comment:
Style/SpaceAroundOperators: Surrounding space missing for operator `=`. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)